Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -727,7 +737,7 @@ namespace AzToolsFramework
                     }
                 }
 
-                if (isInstanceContainerEntity)
+                if (isInFocusTree && !isOwnedByFocusedPrefabInstance)
```

Comment:
We could either feature flag this with a CVAR (to quickly disable it if things go awry) or turn this off for stabilization entirely if we're worried about the behavior, but in my local testing this has only made prefabs work better and more intuitively with focus mode.